ansible/templates/minigraph_dpg.j2

```diff
@@ -96,13 +99,14 @@
         </IPInterface>
         <IPInterface>
           <Name i:Name="true"/>
-{% if 'port-channel' in (vm_topo_config['vm'][vms[index]]['ip_intf']|lower) %}
+{% if 'port-channel' in vm_topo_config['vm'][vms[index]]['ip_intf'][dut_index|int]|lower %}
```

Comment:
Good catch! 
But why are these IPInterfaces configurations are not skipped for supervisor card?
Similarly, there are other configurations like PortChannelInterfaces are not skipped for supervisor card.